code/modules/ruins/objects_and_mobs/ash_walker_den.dm

```diff
@@ -19,13 +19,19 @@
 
 /obj/structure/lavaland/ash_walker/Initialize()
 	.=..()
-	START_PROCESSING(SSprocessing, src)	
+	START_PROCESSING(SSprocessing, src)
 
 /obj/structure/lavaland/ash_walker/deconstruct(disassembled)
 	new /obj/item/assembly/signaler/anomaly (get_step(loc, pick(GLOB.alldirs)))
 	new	/obj/effect/collapse(loc)
 	return ..()
 
+/obj/structure/lavaland/ash_walker/attackby(obj/item/I, mob/living/user, params)
+	. = ..()
+	if(HAS_TRAIT(user, TRAIT_SAVAGE))
+		visible_message("1")
```

Comment:
What does this line of code do (I'm asking because I'm curious, not because I think that it's unneeded)?

Reply:
Ehrm... testing. That shouldn't be there. That code doesn't even work.